### DISCUSSION TITLE
Docker install for largemem kernel

### DISCUSSION THREAD

**Discussion by vg40506** [qdrant/qdrant#4989] [Q&A]
Having issues with Docker install of qdrant on a largemem kernel

**`<jemalloc>: Unsupported system page size`**
Any easy solution?

**Comment by timvisee** [comment]
At this point, no.

You can run it with some manual effort though:
- clone the repository
- remove these lines to disable the use of jemalloc:
  https://github.com/qdrant/qdrant/blob/9fa86106e85d734765efe2c686a3ce975e53fb38/src/main.rs#L53-L58
- compile from source and/or create a Docker image for it

**Reply by Bayzon88** [reply]
could you please explain how to compile from source?

**Reply by timvisee** [reply]
It is described here: https://github.com/qdrant/qdrant/blob/master/docs/DEVELOPMENT.md#build-qdrant
